package.json

```
@@ -11,6 +11,8 @@
     "lint": "next lint",
     "test": "vitest run",
     "test:watch": "vitest",
+    "check:terms": "node ./scripts/check-product-terms.js",
```

Biome (2.5.0)

[error] The key check:terms was already declared.

(lint/suspicious/noDuplicateObjectKeys)